modules/profile/templates/miscweb/site.nginx.erb

```diff
@@ -51,6 +51,12 @@ server {
     include              /etc/nginx/fastcgi_params;
   }
 <%- end -%>
+
+  location / {
```

Comment:
Afaik location blocks are mutually exclusive, so this means this won't apply to `.php` URLs.

It seems `add_header` is valid in server blocks directly as well, if you want to do that for miscweb sites (i.e. above before the first location block). https://nginx.org/en/docs/http/ngx_http_headers_module.html

This is fine to me either way as an incremental start. I am making the assumption that this new catch-all `location` block will inherit the `index` and `try_files` directives from the server block.